src/mobot_client/air_drop_session.py

```diff
@@ -211,55 +196,56 @@ def handle_no_active_airdrop_drop_session(self, customer, message, drop):
         if customer.has_completed_drop(drop):
             self.messenger.log_and_send_message(
                 customer,
-                message.source,
                 ChatStrings.AIRDROP_SUMMARY
             )
 
         elif customer.has_completed_drop_with_error(drop):
             self.messenger.log_and_send_message(
                 customer,
-                message.source,
                 ChatStrings.AIRDROP_INCOMPLETE_SUMMARY
             )
         elif not customer.matches_country_code_restriction(drop):
             self.messenger.log_and_send_message(
                 customer,
-                message.source,
                 ChatStrings.COUNTRY_RESTRICTED
             )
         else:
-            customer_payments_address = self.payments.get_payments_address(message.source)
+            customer_payments_address = self.payments.get_payments_address(customer.source.as_e164)
```

Comment:
TIL about E.164

Reply:
Realistically, turning any of these to strings will default in as_e164